catberry_components/todo/input/index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+module.exports = TodoInput;
+
+/*
+ * This is a Catberry Cat-component file.
+ * More details can be found here
+ * https://github.com/catberry/catberry/blob/master/docs/index.md#cat-components
+ */
+
+/**
+ * Creates new instance of the "todo-input" component.
+ * @constructor
+ */
+function TodoInput() {
+
+}
+
+/**
+ * Input DOM element
+ * @type {Node}
+ */
+TodoInput.prototype.$input = null;
```

Comment:
It is better to rename to `inputNode`, otherwise it is confusing. People can think it is connected with jQuery.